feature/mpls/otg_tests/label_block/README.md

```diff
@@ -0,0 +1,58 @@
+# MPLS-1.1: MPLS label blocks using ISIS
+
+## Summary
+
+Define reserved MPLS label blocks: static and MPLS-SR.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+Topology: ATE1—DUT1
+
+On DUT1 configure:
+
+*   ISIS adjacency between ATE1 and DUT1
+*   Enable MPLS-SR
+*   Static Global Block (start: 1000000 range: 48576)
+*   Segment Routing Global Block (start: 400000 range: 65001)
+*   Segment Routing Local Block (start: 40000 range: 1000)
+
+Verify:
+
+*   Defined blocks are configured on DUT1.
+*   DUT1 advertises its SRGB and SRLB to ATE1.
+
+
+## OpenConfig Path and RPC Coverage
+
+```yaml
+paths:
+  # configuration
+  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/local-id:
+  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/lower-bound:
+  /network-instances/network-instance/mpls/global/reserved-label-blocks/reserved-label-block/config/upper-bound:
+  /network-instances/network-instance/segment-routing/srgbs/srgb/config/local-id:
+  /network-instances/network-instance/segment-routing/srgbs/srgb/config/mpls-label-blocks:
+  /network-instances/network-instance/protocols/protocol/isis/global/segment-routing/config/srgb:
```

Comment:
I am not that familiar with configuring segment routing, but do you also need this path to enable it in ISIS?

[/network-instances/network-instance/protocols/protocol/isis/global/segment-routing/config/enabled](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-network-instance.html#network-instances-network-instance-protocols-protocol-isis-global-segment-routing-config-enabled)

Reply:
Makes sense. Added.